.github/workflows/ci-benchmark.yaml

```diff
@@ -4,7 +4,7 @@ on:
     types:
       - labeled
   schedule:
-    - cron: '0 0 * * *'
+    - cron: '0 0 * * MON'
```

Comment:
Not sure if MON work in GH action cron can you change this to `cron: '0 0 * * 1` or should we run twice in a week then `cron: '0 0 * * 1,3` (for Mon and Wed) @phanikumv @utkarsharma2 WDYT?

Reply:
I'm okay with `0 0 * * 1`